README.md

```diff
@@ -1,2 +1,21 @@
+---
+output: html_document
```

Comment:
Note that this is unnecessary for Markdown documents we intend to view on GitHub. GitHub does all the rendering for us automatically, so there is no need to generate the HTML document. You can generate a preview of this within R Studio, but still don't need to add the `output: html_document` command